llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -471,6 +471,15 @@ class InstrProfSymtab {
 public:
   InstrProfSymtab() = default;
 
+  // Not copyable or movable.
+  // Consider std::unique_ptr for move.
+  // InstrProfSymtab has a few containers as class members, so consider
+  // std::shared_ptr for read-only copy.
```

Comment:
std::shared_ptr has (some) overhead and since we aren't using it in this change, lets drop this comment.